src/templates/pycontw-2020/contents/zh/guidelines.html

```diff
@@ -76,11 +74,12 @@ <h4>PyCon TW 防疫準備</h4>
 </ol>
 
 <h4>個人資料保護聲明</h4>
+
 <ul>
 	<li>
 		PyCon TW 配合防疫期間所蒐集的資料，將在活動日起 28 天後刪除。參與者進出各館與各廳紀錄將作為疫情管控使用，若會後有任何疑似感染情況，PyCon TW
 		將主動通知可能被影響的群眾，且得提供我國衛生主管機關依傳染病防治法等規定進行疫情調查及聯繫使用。詳細聲明請參照：<a
-		href="https://docs.google.com/document/d/1wcTCXs75VxRK2xWdd0lKD57HCMp7ts_MT7lNSJ9_euU/edit#">PyCon Taiwan
+		href="../about/privacy_policy/">PyCon Taiwan
```

Comment:
this indeed functional but we prefer using [`url` tag](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#url) (ref: [warmup page](https://github.com/pycontw/pycon.tw/blob/master/src/templates/pycontw-2020/contents/en/events/warmup-session.html#L9-L11)).

Reply:
got it .